src/inference/Inference.jl

```diff
@@ -431,7 +467,7 @@ include("../contrib/inference/AdvancedSMCExtensions.jl")
 # Typing tools #
 ################
 
-for alg in (:SMC, :PG, :PMMH, :IPMCMC, :MH, :IS, :ESS, :Gibbs)
+for alg in (:SMC, :PG, :PMMH, :IPMCMC, :MH, :MHOld, :IS, :ESS, :Gibbs)
```

Comment:
Is the `MHOld` algorithm still available?

Reply:
Nope, thanks for catching that one.